src/ematlab.ts

```diff
@@ -16,3 +16,7 @@ export function addToPath() {
     }
     core.addPath(path.join(root, "bin"));
 }
+
+export function skipActivationFlag(env: any): string {
```

Comment:
```suggestion
export function skipActivationFlag(env: ProcessEnv): string {
```

or `Dict<string>`... Anything is better than `any`.